### DISCUSSION TITLE
How to hide "using app server https://oooooo.net"

### DISCUSSION THREAD

**Discussion by Card2Me** [ionic-team/capacitor#4497] [Q&A] [answered]
When my android app  is launched, it make a toast message  "using app server https://oooooo.net"

I can not find hide option in documents.

--------------- from capacitors.config.json --------------------
"server": {
    "url": "https://oooooo.net",
    "hostname": "oooooo.net",
    "androidScheme": "https",
    "allowNavigation": ["*"]
  },

**Comment by jcesarmobile** [comment, marked as answer]
That appears when using the url option on debug builds. You can’t remove it unless you build a release build.
We have removed it in capacitor 3, so another option is to update.
